donut/modules/feedback/helpers.py

```diff
@@ -192,31 +193,17 @@ def get_status(group, complaint_id):
     Returns the status of a post or None if complaint_id is invalid
     '''
     res = get_summary(group, complaint_id)
-    return res['status'] if res else None
-
-
-def mark_read(group, complaint_id):
-    '''
-    Sets the status of this complaint to 'read'
-    returns False if complaint_id is invalid
-    '''
-    if get_status(group, complaint_id) is None:
-        return False
-    query = "UPDATE complaint_info SET status = 'read' WHERE complaint_id = %s"
-    with flask.g.pymysql_db.cursor() as cursor:
-        cursor.execute(query, complaint_id)
+    return res['resolved'] if res else None
 
 
-def mark_unread(group, complaint_id):
+def set_status(group, complaint_id, status):
```

Comment:
`set_resolved()`?

Reply:
because `set_status()` to match `get_status()`